tests/e2e.sh

```diff
@@ -26,5 +26,18 @@ function error_exit() {
 
 . ${ROOT}/istio.VERSION || error_exit "Could not source versions"
 
-bazel ${BAZEL_STARTUP_ARGS} run ${BAZEL_RUN_ARGS} //tests/e2e:go_default_test -- ${ARGS} ${@}
-exit $?
+TESTS_TARGETS=($(bazel query 'tests(//tests/e2e/tests/...)'))
+FAILURE_COUNT=0
+SUMMARY='Tests Summary'
+
+for T in ${TESTS_TARGETS[@]}; do
+  bazel ${BAZEL_STARTUP_ARGS} run ${BAZEL_RUN_ARGS} ${T} -- ${ARGS} ${@}
```

Comment:
I think it would be helpful to print a block with the test name, making it clear when we are going to have 20 tests